src/Components/CreateImageWizard/utilities/PasswordValidatedInput.tsx

```diff
@@ -61,14 +62,29 @@ export const PasswordValidatedInput = ({
             />
           </InputGroupItem>
           <InputGroupItem>
-            <Button
-              variant="control"
-              onClick={togglePasswordVisibility}
-              aria-label={isPasswordVisible ? 'Hide password' : 'Show password'}
-              isDisabled={hasPassword && !value}
-            >
-              {isPasswordVisible ? <EyeSlashIcon /> : <EyeIcon />}
-            </Button>
+            {hasPassword && !value ? (
```

Comment:
`hasPassword && !value` could be also put into a constant, which could be reused for disabling the button